src/components/checkbox/index.tsx

```diff
@@ -155,33 +172,38 @@ class Checkbox extends Component<CheckboxPropTypes, CheckboxState> {
   }
 
   render() {
-    const {selectedIcon, color, iconColor, disabled, testID, style, ...others} = this.props;
+    const {selectedIcon, color, iconColor, disabled, testID, label, labelStyle, style, ...others} = this.props;
     return (
-      // @ts-ignore
-      <TouchableOpacity
-        {...this.getAccessibilityProps()}
-        activeOpacity={1}
-        testID={testID}
-        {...others}
-        style={[this.getBorderStyle(), style]}
-        onPress={this.onPress}
-      >
-        {
-          <Animated.View
-            style={[this.styles.container, {backgroundColor: this.getColor()}, {opacity: this.animationStyle.opacity}]}
-          >
-            <Animated.Image
+      <TouchableOpacity row onPress={this.onPress}>
+        <Animated.View
+          {...this.getAccessibilityProps()}
+          activeOpacity={1}
+          testID={testID}
+          {...others}
+          style={[this.getBorderStyle(), style]}
+        >
+          {
+            <Animated.View
               style={[
-                this.styles.selectedIcon,
-                color && {tintColor: iconColor},
-                {transform: this.animationStyle.transform},
-                disabled && {tintColor: DEFAULT_ICON_COLOR}
+                this.styles.container,
+                {backgroundColor: this.getColor()},
+                {opacity: this.animationStyle.opacity}
               ]}
-              source={selectedIcon || Assets.icons.checkSmall}
-              testID={`${testID}.selected`}
-            />
-          </Animated.View>
-        }
+            >
+              <Animated.Image
+                style={[
+                  this.styles.selectedIcon,
+                  color && {tintColor: iconColor},
+                  {transform: this.animationStyle.transform},
+                  disabled && {tintColor: DEFAULT_ICON_COLOR}
+                ]}
+                source={selectedIcon || Assets.icons.checkSmall}
+                testID={`${testID}.selected`}
+              />
+            </Animated.View>
+          }
+        </Animated.View>
+        {label && <Text style={labelStyle}>{label}</Text>}
```

Comment:
I notice that the text is not vertically aligned to the checkbox. 
Also consider adding default spacing to the make the default appearance look good

Reply:
I added the spacing on the private config, but in second thought it is better to apply it in public..